adds freshbooks project documentation (#103)
datamade/how-to: project-management/collateral.md

- **Done/closed**: Issues that have been merged and fixed

Sometimes, project owners will keep a `Done` column in addition to a `Closed` column. In these cases, issues addressed by merged PRs should be moved into `Done` and remain open until sprint planning meetings, when we reflect on them and close them as a group. We typically follow this practice with larger projects like Dedupe.io that involve a substantial budget and time for reflection.

## FreshBooks projects

> Time is tracked at DataMade using [FreshBooks](https://my.freshbooks.com). For each item in the scope of work, a new FreshBooks [project](https://my.freshbooks.com/#/projects) should be created.
> 
> Project setup and formatting:

Reviewer: ```suggestion
Time is tracked at DataMade using [FreshBooks](https://my.freshbooks.com). For each item in the scope of work, a new FreshBooks [project](https://my.freshbooks.com/#/projects) should be created with the following configuration.
```

* Name: Follows the convention of `Project title: Scope item title` (Example: `Payroll Phase 3: Meetings and Project Planning`)

> * Client: Organization, name and email of the client contact. If they are a new client, you may have to create them in Freshbooks first.

Author: I typically make clients when creating a new project. All that's really required is name, company, and email

<img width="1152" alt="Screen Shot 2020-09-10 at 9 51 36 AM" src="https://user-images.githubusercontent.com/919583/92748980-4a211400-f34b-11ea-8713-1eb5eb1c3eb3.png">

* Type: Hourly project with an hourly staff rate. Using a flat rate type will set all tracked time to non-billable.
* Project members: All staff assigned to the project. Note, if a staff member is not assigned to the project, it will not show up in their list of projects to track time against.
* Project services:
* Hourly budget: Number of hours estimated for this line item

> * Billing method: Team member rates. Team member rates are set based staff position.

Reviewer: I got confused here. It'd be easier to stay consistent and call this what it's labeled in the interface.

```suggestion
Set hourly rates: Under "Select a billing method", select "Team Member Rates". Team member rates are set based on position. Default rates have been set for each team member in FreshBooks, but can be altered based on project in this screen.
```

Would you also move this up next to Project Type, so these points are in the same order as they appear in the UI?

* End date: Optional. We don't track this in Freshbooks.

Here's an example for the CalFWD project:

![CalFWD project in FreshBooks](freshbooks-calfwd.png)

> FreshBooks has [documentation on how projects work here](https://www.freshbooks.com/support/which-billing-method-should-i-choose).

Reviewer: ```suggestion
For further reference, FreshBooks has [documentation on how projects work here](https://www.freshbooks.com/support/which-billing-method-should-i-choose).
```

    - If your project is for a new client, you can add them to FreshBooks from the project creation screen. Simply fill in their information in the Client form, and they will be added to the system. If you aren't sure who the client contact is, consult a partner or lead developer.
- **Project members**: Select all staff assigned to the project. Note, if a staff member is not assigned to the project, it will not show up in their list of projects to track time against.
- **Project type**: Always select `Hourly Project`. **Do not use `Flat Rate Project`, as it will set all tracked time to non-billable.**
- **Set hourly rates**: Under "Select a billing method", select "Team Member Rates". Team member rates are set based on position. Default rates have been set for each team member in FreshBooks, but can be altered based on project in this screen.
- **Project services**: Select relevant services from the existing list. `Research`, `Development`, `Project Management`, and `Meetings` is a good starting point for most projects.

> - **Hourly budget**: Divide the cost of the scope item by $175 to estimate the number of hours budgeted for this scope item.

Reviewer: @derekeder When I write scopes, I always start with hours and multiply by $175 before sending. Is this a team-wide practice, i.e., does this point make sense? Otherwise, we'll need to revise.

Author: yes, this is how we price out scopes, so we should do the same in freshbooks
